src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1226,7 +1216,7 @@ ExpressionPtr Visitor::visitTypesafe(Parser::IriOrFunctionContext* ctx) {
 
 // ____________________________________________________________________________________
 std::string Visitor::visitTypesafe(Parser::RdfLiteralContext* ctx) {
-  string ret = ctx->string()->getText();
+  string ret = visitTypesafe(ctx->string());
```

Comment:
TODO: This should really be an `RdfLiteral` class that stores a unified version of the string, and the langtag/datatype separately.
(The comment is enough for now, but we have to remeber these things)